src/constants/DeviceFrames.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2023 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
```

Comment:
Could you maybe add a comment here to explain that this is empty because swift-docc-render does not provide any actual device frames on its own by default?

Reply:
yep yep.

Reply:
```suggestion

// Swift-DocC-Render does not provide device frames out of the box.
```